frontend/src/App.jsx

```diff
@@ -4,6 +4,7 @@ import Navbar from "./components/Navbar/Navbar";
 import Home from "./page/Home/Home";
 import Footer from "./components/Footer/Footer";
 // import Author from "./components/Author/Author";
+// import Gallery from "./page/Home/Galerie/Galerie";
 import Carousel from "./components/Carousel/carousel";
```

Comment:
Death code

Reply:
same thing - i will call the gallery the next time i use the code so to avoid rewriting code, ill just comment it out for the moment of the push

Reply:
Please delete if is not use.